docs/src/templates/sandbox/index.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from "react";
+import { LiveProvider, LiveEditor } from "react-live";
+import dracula from "prism-react-renderer/themes/dracula";
+import styled, { css, createGlobalStyle } from "styled-components";
+import { ThemeProvider, defaultTheme, Stack } from "@kiwicom/orbit-components";
+
+import { getModules, copyImports } from "../../components/ReactExample/helpers";
+import Board from "../../components/ReactExample/Board";
+import ViewportsRuler from "../../components/ReactExample/ViewportsRuler";
+import Frame from "../../components/ReactExample/Frame";
+import useSandbox from "./useSandbox";
+
+const StyledWrapper = styled.div`
+  display: grid;
+  min-height: 100%;
+  grid-template-rows: auto 1fr auto;
+  grid-template-columns: 100%;
+`;
+
+const StyledPreviewWrapper = styled.div<{ width: number }>`
+  ${({ theme, width }) => css`
+    max-width: ${width}px;
+    width: 100%;
+    height: 100%;
+    overflow: auto;
+    padding: ${theme.orbit.spaceXLarge} ${theme.orbit.spaceMedium} ${theme.orbit.spaceXSmall};
+  `};
+`;
+
+const GlobalStyle = createGlobalStyle`
+  body,
+  html,
+  #___gatsby,
+  #gatsby-focus-wrapper {
+    height: 100%;
+    width: 100%;
+  }
+`;
+
+const Sandbox = ({ pathContext }) => {
+  const { example, id, example_id, scope } = pathContext;
+  const [isEditorOpened, setOpenEditor] = React.useState(true);
+  const [width, setPreviewWidth] = React.useState(0);
+  const { code, origin } = useSandbox(example_id);
+
+  const getCurrentWidth = React.useCallback(size => setPreviewWidth(size), []);
```

Comment:
`getCurrentWidth` doesn't get a width, it sets it, I recommend a name like `handleChangeRulerSize`.